contracts/governance/StakingRewards/StakingRewardsProxy.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity ^0.5.17;
+
+import "./StakingRewardsStorage.sol";
+import "../../proxy/UpgradableProxy.sol";
+
+/**
+ * @title StakingRewards Proxy contract.
+ * @dev StakingRewards contract should be upgradable. Used UpgradableProxy.
+ * StakingRewardsStorage is deployed with the upgradable functionality
+ * by using this contract instead, that inherits from UpgradableProxy with
+ * the possibility of being enhanced and re-deployed.
+ * */
+contract StakingRewardsProxy is StakingRewardsStorage, UpgradableProxy {
```

Comment:
`UpgradableProxy` contract is similar to EIP-1967 which means that it would be better if the proxy does not know about the logic of `StakingRewards` and its storage

Reply:
we prefer to inherit from the storage for 2 reasons:
1) it is initialized with default values on the proxy deployment (can also be used initialization from logic, but inheritance looks like a more clear way to do it as in this version of the proxy there is no initialization call on new logic)
2) for clarity on the actual storage source and structure
because storage does not contain any functions, proxy knows nothing about the logic but only about the structure
are there any other caveats you see @korepkorep if we leave it as is?